Known_Issues/All VMMS operations are blocked on Silver or higher durability.md

```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS 
+- The node being deleted (seed node) is stuck in Disabling state
+
+## Symptoms
+- One of the seed nodes is stuck disabling with Safety Check - **EnsureSeedNodeQuorum**
+- All operation started after the delete operation will be blocked until the deletion completes, including adding new VMs
+
+    ![Node Deactivation Info](../media/NodeDeactivationInfo1.png)
+
+## Cause ##
+
+**Service Fabric is blocking deletion of VM to prevent violating configured ReliabilityLevel**
+
+- The current delete operation, initiated by the customer through VMSS, is trying to reduce the number of allocated nodes below the minimum number nodes required for the cluster reliability level for Primary node type.
+  
+  - See [Reliability Charactaristics of the Cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity#the-reliability-characteristics-of-the-cluster) for number of nodes required for each reliability level
+
+**Customer has to determine the root cause of why the VM is being deleted**
+
+- The VMSS operation that resulted in the specific instance is being deleted / deallocated - Audit logs for VMSS in Portal should have the information  
+- Most common reasons  
+  - Customer initiated delete / deallocation from portal or CLI
+    - normally to address some other issue and as a result put the cluster into worse state
+  - Auto scale settings where min is < number of nodes required for reliability level
+  - Some automation run by customer that is initiating the delete
+
+**NOTE:** Deallocate operations on instances in scaleset associated with SF is not supported.
+
+## Example ##
+
+ . | .
+----------|----------
+Current Reliability Level | Silver
+Seed nodes required for current reliability level | 5
+Current number of nodes of Primary node type | 5
+Does removing a node violate reliability level | Yes
+
+## Mitigation ##
+
+**The following is the only safe mitigation**
+
+**NOTE:** Mitigation time will take at a minimum 60-90 minutes. Depending on the configuration of SF cluster upgrade settings it can take significantly longer.  Please open a support ticket through the Azure Portal for assistance.
```

Comment:
Suggestion, slight tweak as we are pushing for self-service?

>Please open a support ticket through the Azure Portal for assistance.
Please open a support ticket through the Azure Portal if you need assistance. (link to support page?)